cmd/influxd/launcher/pkger_test.go

```diff
@@ -793,7 +756,95 @@ spec:
 				))
 				require.NoError(t, err)
 
-				verifyCompleteSummary(t, newPkg.Summary(), true)
+				sum := newPkg.Summary()
+
+				labels := sum.Labels
+				require.Len(t, labels, 2)
+				assert.Equal(t, "label_1", labels[0].Name)
+				assert.Equal(t, "the 2nd label", labels[1].Name)
+
+				bkts := sum.Buckets
+				require.Len(t, bkts, 1)
+				assert.NotEmpty(t, bkts[0].PkgName)
+				assert.Equal(t, "rucketeer", bkts[0].Name)
+				hasLabelAssociations(t, bkts[0].LabelAssociations, 2, "label_1", "the 2nd label")
+
+				checks := sum.Checks
+				require.Len(t, checks, 2)
+				assert.Equal(t, "check 0 name", checks[0].Check.GetName())
+				hasLabelAssociations(t, checks[0].LabelAssociations, 1, "label_1")
+				assert.Equal(t, "check_1", checks[1].Check.GetName())
+				hasLabelAssociations(t, checks[1].LabelAssociations, 1, "label_1")
+
+				dashs := sum.Dashboards
+				require.Len(t, dashs, 1)
+				assert.NotEmpty(t, dashs[0].Name)
+				assert.Equal(t, "dash_1", dashs[0].Name)
+				assert.Equal(t, "desc1", dashs[0].Description)
```

Comment:
had to separate apply/export all assertions for each test. To kludgy to keep them together